src/components/Todo.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from './User';
+
+export function Todo({ title, completed, user }) {
+  return (
+    <>
+      <div>
+        <User {...user} />
+        <p>{title}</p>
+        <p>{completed ? 'true' : 'false'}</p>
```

Comment:
```suggestion
        <p>{completed ? 'Completed' : 'In progress'}</p>
```
Что `true`? Что `false`?